tasks/start.yml

```diff
@@ -3,7 +3,10 @@
 
 - name: Wait for Jenkins to start
   uri:
-    url: "{{ jenkins_url }}"
+    url: "{{ jenkins_url_health_check }}"
+    url_username: "{{ jenkins_health_check_user | default(omit) }}"
+    url_password: "{{ jenkins_health_check_password | default(omit) }}"
+    force_basic_auth: "{{ 'true' if (jenkins_health_check_user) else 'false' }}"
```

Comment:
I have a feeling that this caused the test failure at https://travis-ci.org/emmetog/ansible-jenkins/builds/617611371#L654. This is a bit verbose, but maybe it could be solved by something along the lines of:

```yaml
- name: Initialize jenkins_force_basic_auth parameter
  set_fact:
    jenkins_force_basic_auth: false

- name: Set jenkins_force_basic_auth parameter
  set_fact:
    jenkins_force_basic_auth: true
  when: jenkins_health_check_user is defined

- name: Wait for Jenkins to start
  uri:
    force_basic_auth: "{{ jenkins_force_basic_auth }}"
    # ....
```